### DISCUSSION TITLE
v0.10.4.1

### DISCUSSION THREAD

**Discussion by AsamK** [AsamK/signal-cli#902] [Releases]
**Full Changelog**: https://github.com/AsamK/signal-cli/compare/v0.10.4...v0.10.4.1

<hr /><em>This discussion was created from the release <a href='https://github.com/AsamK/signal-cli/releases/tag/v0.10.4.1'>v0.10.4.1</a>.</em>

**Comment by ghost** [comment]
What am I doing wrong?

[sig@vmi805292 bin]$ ./signal-cli
Error: A JNI error has occurred, please check your installation and try again
Exception in thread "main" java.lang.UnsupportedClassVersionError: org/asamk/signal/Main has been compiled by a more recent version of the Java Runtime (class file version 61.0), this version of the Java Runtime only recognizes class file versions up to 52.0

[sig@vmi805292 bin]$ java -version
openjdk version "1.8.0_322"
OpenJDK Runtime Environment (build 1.8.0_322-b06)
OpenJDK 64-Bit Server VM (build 25.322-b06, mixed mode)

**Reply by AsamK** [reply]
You need Java 17

**Reply by ghost** [reply]
Thanks a lot, works like a charm!
